generate_config.py

```diff
@@ -173,6 +174,9 @@
 /etc/nginx/ssl/""" + hostname + """.crt; # Wildcard SSL certificate
 /etc/nginx/ssl/""" + hostname + """.key; # Wildcard SSL key
 
+Note: by default self-registration is enabled. You can disable this by changing the 'self_registration' option in the config file to 
```

Comment:
Yeah I would have thought so too, but it seems to work just fine. Yaml is a bit weird (I've never used it before personally), and I think PyYAML converts "yes/no" into Booleans (see https://stackoverflow.com/questions/36463531/pyyaml-automatically-converting-certain-keys-to-boolean-values).